updatedJavaScriptCode.js

```diff
@@ -0,0 +1,364 @@
+//1- babel-cli and babel-preset-env were installed
+//2- create a file .babelrc with content {    "presets": ["env"]}
+//3- change from require to import
+//4- run npx babel-node filename.js
+import constants from './constantfile';
```

Comment:
the file is too big , we need to separate it into parts. make a folder and separate each into a part. This will make it more readable and better.

Reply:
done, and i started with the typescipt part